README.md

```diff
@@ -61,7 +61,7 @@ the Cat class, giving us a class which supports stubbing and verifying.
 
 [NULL_SAFETY_README]: https://github.com/dart-lang/mockito/blob/master/NULL_SAFETY_README.md
 
-## Let's verify some behaviour!
```

Comment:
This is just the non-American spelling of the word. Not sure which our preference should be...